library/src/ua.rs

```diff
@@ -84,7 +84,7 @@ impl UserAgent for UA {
 
             // Chrome and Opera on iOS uses a UIWebView of the underlying platform to render content. By stripping the CriOS or OPiOS strings, the useragent parser will alias the user agent to ios_saf for the UIWebView, which is closer to the actual renderer
             let ua_string = Regex::new(
-                r"(?i)((CriOS|OPiOS)\/(\d+)\.(\d+)\.(\d+)\.(\d+)|(FxiOS\/(\d+)\.(\d+)))",
+                r"(?i)((CriOS|OPiOS)\/(\d+)\.(\d+)\.(\d+)\.(\d+)|(FxiOS\/(\d+)\.(\d+))) ",
```

Comment:
I don't love ` ` (a single space), but it's more specific than `\s`, so perhaps better?